web/package.json

```diff
@@ -38,8 +38,8 @@
     "typescript": "^4.7.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "DISABLE_ESLINT_PLUGIN=true react-scripts start",
```

Comment:
Do we really want it disabled on yarn start?

Reply:
This is a per-developer preference. But you'r right, most will likely like to keep this. As long as it still compiles with eslint errors. I will reenable that.